src/DiagnosticsManager.ts

```diff
@@ -40,18 +43,23 @@ export class DiagnosticsManager implements vscode.Disposable {
 
     private diagnosticCollection: vscode.DiagnosticCollection =
         vscode.languages.createDiagnosticCollection("swift");
+    private allDiagnostics: Map<string, vscode.Diagnostic[]> = new Map();
 
     constructor(context: WorkspaceContext) {
         this.onDidChangeConfigurationDisposible = vscode.workspace.onDidChangeConfiguration(e => {
             if (e.affectsConfiguration("swift.diagnosticsCollection")) {
-                if (!this.includeSwiftcDiagnostics()) {
-                    // Clean up "swiftc" diagnostics
-                    this.removeSwiftcDiagnostics();
-                }
-                if (!this.includeSourceKitDiagnostics()) {
-                    // Clean up SourceKit diagnostics
-                    this.removeSourceKitDiagnostics();
-                }
+                // if (!this.includeSwiftcDiagnostics()) {
+                //     // Clean up "swiftc" diagnostics
+                //     this.removeSwiftcDiagnostics();
+                // }
+                // if (!this.includeSourceKitDiagnostics()) {
+                //     // Clean up SourceKit diagnostics
+                //     this.removeSourceKitDiagnostics();
+                // }
```

Comment:
nit: should just delete this instead of commenting it out

Reply:
done